Refactoring for Android Studio project

ragomusic opened the pull request:
Created Android Studio project and moved files around to fit in the
Android Studio project schema.

Note that android.mk files will not work for now.

gkasten commented:
Hi Ricardo, I'm surprised this was needed.  I was working with Android
Studio without the renaming.
I will share my build instructions, and compare with yours.

On Mon, Mar 23, 2015 at 3:14 PM, ragomusic <notifications@github.com> wrote:

> Created Android Studio project and moved files around to fit in the
> Android Studio project schema.
>
> Note that android.mk files will not work for now.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/gkasten/drrickorang/pull/1
> Commit Summary
>
>    - Refactoring for Android Studio project
>
> File Changes
>
>    - *A* LoopbackApp/.idea/.name
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-0> (1)
>    - *A* LoopbackApp/.idea/compiler.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-1> (23)
>    - *A* LoopbackApp/.idea/copyright/profiles_settings.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-2> (3)
>    - *A* LoopbackApp/.idea/dictionaries/rago.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-3> (3)
>    - *A* LoopbackApp/.idea/encodings.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-4> (5)
>    - *A* LoopbackApp/.idea/gradle.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-5> (18)
>    - *A* LoopbackApp/.idea/misc.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-6> (23)
>    - *A* LoopbackApp/.idea/modules.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-7> (10)
>    - *A* LoopbackApp/.idea/scopes/scope_settings.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-8> (5)
>    - *A* LoopbackApp/.idea/vcs.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-9> (7)
>    - *A* LoopbackApp/LoopbackApp.iml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-10> (19)
>    - *A* LoopbackApp/app/app.iml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-11> (90)
>    - *A* LoopbackApp/app/build.gradle
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-12> (24)
>    - *R* LoopbackApp/app/src/main/AndroidManifest.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-13> (2)
>    - *A* LoopbackApp/app/src/main/java/org/drrickorang/.DS_Store
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-14> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/AudioFileOutput.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-15> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackActivity.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-16> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackApplication.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-17> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackAudioThread.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-18> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/NativeAudioThread.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-19> (0)
>    - *R* LoopbackApp/app/src/main/java/org/drrickorang/loopback/Pipe.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-20> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/SettingsActivity.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-21> (0)
>    - *R*
>    LoopbackApp/app/src/main/java/org/drrickorang/loopback/WavePlotView.java
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-22> (0)
>    - *R* LoopbackApp/app/src/main/jni/Android.mk
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-23> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/atomic.c
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-24> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/atomic.h
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-25> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/fifo.c
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-26> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/fifo.h
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-27> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/roundup.c
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-28> (0)
>    - *R* LoopbackApp/app/src/main/jni/audio_utils/roundup.h
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-29> (0)
>    - *R* LoopbackApp/app/src/main/jni/jni_sles.c
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-30> (0)
>    - *R* LoopbackApp/app/src/main/jni/jni_sles.h
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-31> (0)
>    - *R* LoopbackApp/app/src/main/jni/sles.cpp
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-32> (0)
>    - *R* LoopbackApp/app/src/main/jni/sles.h
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-33> (0)
>    - *R* LoopbackApp/app/src/main/res/drawable-xxxhdpi/ic_launcher.png
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-34> (0)
>    - *R* LoopbackApp/app/src/main/res/layout/main_activity.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-35> (0)
>    - *R* LoopbackApp/app/src/main/res/layout/settings_activity.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-36> (0)
>    - *R* LoopbackApp/app/src/main/res/values/strings.xml
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-37> (0)
>    - *A* LoopbackApp/build.gradle
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-38> (15)
>    - *A* LoopbackApp/gradle/wrapper/gradle-wrapper.jar
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-39> (0)
>    - *A* LoopbackApp/gradle/wrapper/gradle-wrapper.properties
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-40> (6)
>    - *A* LoopbackApp/gradlew
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-41> (164)
>    - *A* LoopbackApp/gradlew.bat
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-42> (90)
>    - *A* LoopbackApp/settings.gradle
>    <https://github.com/gkasten/drrickorang/pull/1/files#diff-43> (1)
>
> Patch Links:
>
>    - https://github.com/gkasten/drrickorang/pull/1.patch
>    - https://github.com/gkasten/drrickorang/pull/1.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gkasten/drrickorang/pull/1>.
>

ragomusic commented:
Hi Glenn.
You were copying the files every time (importing), so, changes made to the
files would be lost.
I'm using Android Studio for all the development, including UI, etc.

BTW, I found a bunch of little bugs on the JAVA version I already fixed,
and a likely problem in native for older (less memory) devices, but this
one is trickier.

-rago

On Tue, Mar 24, 2015 at 3:10 PM, gkasten <notifications@github.com> wrote:

> Hi Ricardo, I'm surprised this was needed. I was working with Android
> Studio without the renaming.
> I will share my build instructions, and compare with yours.
>
> On Mon, Mar 23, 2015 at 3:14 PM, ragomusic <notifications@github.com>
> wrote:
>
> > Created Android Studio project and moved files around to fit in the
> > Android Studio project schema.
> >
> > Note that android.mk files will not work for now.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/gkasten/drrickorang/pull/1
> > Commit Summary
> >
> > - Refactoring for Android Studio project
> >
> > File Changes
> >
> > - *A* LoopbackApp/.idea/.name
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-0> (1)
> > - *A* LoopbackApp/.idea/compiler.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-1> (23)
> > - *A* LoopbackApp/.idea/copyright/profiles_settings.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-2> (3)
> > - *A* LoopbackApp/.idea/dictionaries/rago.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-3> (3)
> > - *A* LoopbackApp/.idea/encodings.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-4> (5)
> > - *A* LoopbackApp/.idea/gradle.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-5> (18)
> > - *A* LoopbackApp/.idea/misc.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-6> (23)
> > - *A* LoopbackApp/.idea/modules.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-7> (10)
> > - *A* LoopbackApp/.idea/scopes/scope_settings.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-8> (5)
> > - *A* LoopbackApp/.idea/vcs.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-9> (7)
> > - *A* LoopbackApp/LoopbackApp.iml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-10> (19)
> > - *A* LoopbackApp/app/app.iml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-11> (90)
> > - *A* LoopbackApp/app/build.gradle
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-12> (24)
> > - *R* LoopbackApp/app/src/main/AndroidManifest.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-13> (2)
> > - *A* LoopbackApp/app/src/main/java/org/drrickorang/.DS_Store
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-14> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/AudioFileOutput.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-15> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackActivity.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-16> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackApplication.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-17> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/LoopbackAudioThread.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-18> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/NativeAudioThread.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-19> (0)
> > - *R* LoopbackApp/app/src/main/java/org/drrickorang/loopback/Pipe.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-20> (0)
> > - *R*
> >
> LoopbackApp/app/src/main/java/org/drrickorang/loopback/SettingsActivity.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-21> (0)
> > - *R*
> > LoopbackApp/app/src/main/java/org/drrickorang/loopback/WavePlotView.java
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-22> (0)
> > - *R* LoopbackApp/app/src/main/jni/Android.mk
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-23> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/atomic.c
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-24> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/atomic.h
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-25> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/fifo.c
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-26> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/fifo.h
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-27> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/roundup.c
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-28> (0)
> > - *R* LoopbackApp/app/src/main/jni/audio_utils/roundup.h
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-29> (0)
> > - *R* LoopbackApp/app/src/main/jni/jni_sles.c
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-30> (0)
> > - *R* LoopbackApp/app/src/main/jni/jni_sles.h
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-31> (0)
> > - *R* LoopbackApp/app/src/main/jni/sles.cpp
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-32> (0)
> > - *R* LoopbackApp/app/src/main/jni/sles.h
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-33> (0)
> > - *R* LoopbackApp/app/src/main/res/drawable-xxxhdpi/ic_launcher.png
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-34> (0)
> > - *R* LoopbackApp/app/src/main/res/layout/main_activity.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-35> (0)
> > - *R* LoopbackApp/app/src/main/res/layout/settings_activity.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-36> (0)
> > - *R* LoopbackApp/app/src/main/res/values/strings.xml
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-37> (0)
> > - *A* LoopbackApp/build.gradle
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-38> (15)
> > - *A* LoopbackApp/gradle/wrapper/gradle-wrapper.jar
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-39> (0)
> > - *A* LoopbackApp/gradle/wrapper/gradle-wrapper.properties
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-40> (6)
> > - *A* LoopbackApp/gradlew
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-41> (164)
> > - *A* LoopbackApp/gradlew.bat
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-42> (90)
> > - *A* LoopbackApp/settings.gradle
> > <https://github.com/gkasten/drrickorang/pull/1/files#diff-43> (1)
> >
> > Patch Links:
> >
> > - https://github.com/gkasten/drrickorang/pull/1.patch
> > - https://github.com/gkasten/drrickorang/pull/1.diff
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/gkasten/drrickorang/pull/1>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gkasten/drrickorang/pull/1#issuecomment-85712693>.
>
